Zend/zend_compile.h

```diff
@@ -209,7 +209,7 @@ typedef struct _zend_oparray_context {
 /* Static method or property                              |     |     |     */
 #define ZEND_ACC_STATIC                  (1 <<  4) /*     |  X  |  X  |     */
 /*                                                        |     |     |     */
-/* Final class or method                                  |     |     |     */
+/* Final class, property, or method                       |     |     |     */
 #define ZEND_ACC_FINAL                   (1 <<  5) /*  X  |  X  |     |     */
```

Comment:
Update the visual representation in the table as well?
```c
/* Class, property and method flags                  class|meth.|prop.|const*/	
```

```suggestion
#define ZEND_ACC_FINAL                   (1 <<  5) /*  X  |  X  |  X  |     */
```

Reply:
I would rather not update it for now because a new flag will be added for sure if the RFC is accepted (unless it's ok if I use the readonly keyword with `ZEND_ACC_FINAL` - but I think it's not).